index.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, user-scalable=no"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <div class="container">
+          <div class="headerInner">
+            <div class="logo">
+              <img src="image/skoutTop.png" alt="Skout" class="skoutTop"/>
```

Comment:
In this case you can use a text for logo and you don't need to put second grey image

Reply:
done